bundle/default-bundle/src/test/resources/application.properties

```diff
@@ -1,31 +1,26 @@
 server.port=8080
-zeebe.client.broker.gateway-address=localhost:26500
-zeebe.client.security.plaintext=true
-
-# Operate config for use with docker-compose.yml
-camunda.operate.client.url=http://localhost:8081
-#camunda.operate.client.username=demo
-#camunda.operate.client.password=demo
 
 management.context-path=/actuator
 management.endpoints.web.exposure.include=metrics,health,prometheus
 management.endpoint.health.group.readiness.include[]=zeebeClient,operate
 management.endpoint.health.show-components=always
 management.endpoint.health.show-details=always
 
-camunda.identity.type=keycloak
-camunda.identity.base-url=http://localhost:8084
-camunda.identity.issuer=http://localhost:18080/auth/realms/camunda-platform
-camunda.identity.issuer-backend-url=http://localhost:18080/auth/realms/camunda-platform
-camunda.identity.audience=connectors
-camunda.identity.client-id=connectors
-camunda.identity.client-secret=XALaRPl5qwTEItdwCMiPS62nVpKs7dL7
-
-debug=true
+camunda.client.operate.base-url=http://localhost:8081
+camunda.client.zeebe.gateway-url=http://localhost:26500
+camunda.client.zeebe.defaults.max-jobs-active=32
+camunda.client.zeebe.defaults.worker-threads=10
 
-zeebe.client.worker.max-jobs-active=32
-zeebe.client.worker.threads=10
+# Config for use with docker-compose.yml
+camunda.client.mode=oidc
+camunda.client.auth.client-id=connectors
+camunda.client.auth.client-secret=XALaRPl5qwTEItdwCMiPS62nVpKs7dL7
+camunda.client.auth.oidc-type=keycloak
+camunda.client.auth.issuer=http://localhost:18080/auth/realms/camunda-platform
+camunda.client.identity.audience=connectors
+camunda.client.identity.base-url=http://localhost:8084
```

Comment:
I switched to the new property config because there seems to be an issue with the old one (it refuses to start without additional explicit Zeebe client configuration via env variables, i.e. zeebe client auth is not configured by spring zeebe with the old properties format).

However, the old property format still works (although deprecated), and the `docker-compose` files also work without any adjustments (because there we inject zeebe config via env variables, which solves the problem described above).

To discuss whether we want to replace the property config in `docker-compose`, Helm charts, and SaaS config.

Reply:
We will keep the configuration in Docker compose / Helm / SaaS as is